benchmark/pom.xml

```diff
@@ -44,7 +44,12 @@
         <dependency>
             <groupId>io.prometheus</groupId>
             <artifactId>simpleclient</artifactId>
-            <version>0.6.1-SNAPSHOT</version>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.prometheus</groupId>
+            <artifactId>simpleclient_hdrhistogram</artifactId>
+            <version>${project.version}</version>
```

Comment:
```
[ERROR] Failed to execute goal on project benchmarks: Could not resolve dependencies for project io.prometheus:benchmarks:jar:0.6.1-SNAPSHOT: Could not find artifact io.prometheus:simpleclient_hdrhistogram:jar:0.6.1-SNAPSHOT in sonatype-nexus-snapshots (https://oss.sonatype.org/content/repositories/snapshots)
```

I ran into the same issue with IntelliJ IDEA locally, it was trying to resolve nonexistent artifacts.
I fixed it by using `${project.version}` instead of `0.6.1-SNAPSHOT` for dependencies between modules.
It looks like the same fix doesn't work for CicleCI, but maybe there's a better way to handle this.

Reply:
The CI changed recently, so it's possible it doesn't handle it well. Let's worry about it at merge time.